### ISSUE TITLE
Suggestion: Incoming tx warning

### ISSUE THREAD

**Issue by Raganius** [iotaledger/android-wallet-app#12]
Hey, there!

Thanks for this great app. I loved the new feature of warnings (in my device, vibration) when the PoW is finished, and from this feature I have a suggestion: It would be convenient if the wallet warns the user (vibrates/beeps) whenever there is a new transaction (or message) arrived.

Thank you again. Keep up the good work!

**Comment by dthorpe** [comment]
Great suggestion, but I'd rephrase it as a "notification" rather than "warning".

"Warning! You're receiving coins!" sounds kinda silly.  :)
